packages/arg-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mongosh/arg-parser",
-  "version": "0.0.0-dev.0",
+  "version": "2.3.7",
```

Comment:
these were all set to 2.3.7 manually to get the current repository consistent with the latest release